openfe/protocols/openmm_afe/equil_afe_settings.py

```diff
@@ -40,20 +40,52 @@
 class AlchemicalSettings(SettingsBaseModel):
     """Settings for the alchemical protocol
 
-    These settings describe the lambda schedule and the creation of the
-    hybrid system.
+    Empty place holder for right now.
     """
 
-    lambda_elec_windows = 12
-    """Number of lambda electrostatic alchemical steps, default 12"""
-    lambda_vdw_windows = 12
-    """Number of lambda vdw alchemical steps, default 12"""
 
-    @validator('lambda_elec_windows', 'lambda_vdw_windows')
-    def must_be_positive(cls, v):
-        if v <= 0:
-            errmsg = ("Number of lambda steps must be positive ")
-            raise ValueError(errmsg)
+class LambdaSettings(SettingsBaseModel):
+    """Settings for lambda schedule
+    """
+    lambda_elec: list = [0.0, 0.25, 0.5, 0.75, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0,
+                         1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
+    """
+    List of floats of lambda values for the electrostatics. 
+    Zero means state A and 1 means state B.
+    """
+    lambda_vdw: list = [0.0, 0.0, 0.0, 0.0, 0.0, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5,
+                        0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95, 1.0]
+    """
+    List of floats of lambda values for the van der Waals.
+    """
+    lambda_restraints: list = None
+    """
+    List of floats of lambda values for the restraints.
+    """
+
+    @validator('lambda_elec', 'lambda_vdw', 'lambda_restraints')
+    def must_be_between_0_and_1(cls, v):
```

Comment:
worth validating that there is actually a 0 and 1.0 value?

Reply:
I would say that (esp. for the restraints) it is ok if there's only a 0 value, or only 1. One may e.g. also only want to turn off electrostatic interactions, while keeping the vdw. But we should probably add more validators/warnings here, e.g. not allowing/warning about turning off vdw while electrostatics are still turned on. Thoughts @IAlibay ?

Reply:
After our discussion I will be adding validators for checking naked charges and monotonical schedules.